libraries/typings-generator/src/StringValuesTypingsGenerator.ts

```diff
@@ -42,11 +42,22 @@ export interface IExportAsDefaultOptions {
    */
   interfaceName?: string;
 
+  /**
+   * @deprecated - Use {@link IExportAsDefaultOptions.interfaceDocumentationComment} instead.
+   */
+  documentationComment?: string;
```

Comment:
What's the plan for deprecation of this? Removal on the next major version bump?

Reply:
Yup